codespell_lib/data/dictionary.txt

```diff
@@ -2961,6 +2970,8 @@ autenticated->authenticated
 autenticates->authenticates
 autenticating->authenticating
 autentication->authentication
+autenticatior->authenticatior
```

Comment:
Is `authenticatior` a realy word? Do you mean `authenticator` (minus the last `i`)?

Reply:
Ah, yes. I completely messed this up, thanks a lot for the review / catching this :+1: 

Fixed in the recent push.